package.json

```diff
@@ -22,9 +22,11 @@
     "url": "https://github.com/karlhorky/react-malarkey/issues"
   },
   "homepage": "https://github.com/karlhorky/react-malarkey#readme",
+  "dependencies": {
+    "malarkey": "^1.3.3"
```

Comment:
Okay, I'm fine with leaving this in `dependencies` for now since it's probably an edge case (an instance of `malarkey` in a project which also has `react-malarkey`). I'll keep an eye on this though - if someone opens an issue I'll switch it back to `peerDependencies` with a note in the readme to install `malarkey` as well.